src/components/Contentful/JourneyCardCarousel.vue

```diff
@@ -246,9 +251,11 @@ const orderedSlides = computed(() => {
 	}
 
 	if (props.slidesNumber) {
-		sortedSlides = sortedSlides.slice(0, props.slidesNumber);
+		const slidesNumber = props.isGoalComplete && props.inLendingStats ? props.slidesNumber + 1 : props.slidesNumber;
```

Comment:
Why is this needed?

Reply:
So is this logic leaving goal card + 2 achievements?

Reply:
The ticket mentions replacing goals card with either:

1. top achievement (using old sort rules) + countries
2. top 3 achievements (again old sort rules) => all countries lent to

It looks like this PR might not be doing this?